src/dioptra/restapi/v1/workflows/schema.py

```diff
@@ -19,6 +19,8 @@
 
 from marshmallow import Schema, fields
 
+STRING_DEFAULT = "<string>"
+
```

Comment:
This doesn't seem like its being used. If its not used anywhere, it can be removed.